translations/extension-runtime.json

```diff
@@ -3193,7 +3193,45 @@
         "stretch@_y stretch": "y拉伸",
         "true-fantom/base@_Base": "进制转换",
         "true-fantom/couplers@_Couplers": "耦合器",
-        "true-fantom/regexp@_RegExp": "正则表达式"
+        "true-fantom/regexp@_RegExp": "正则表达式",
+        "Gandi/LayerManager@extensionName": "图层管理",
+        "Gandi/LayerManager@div1": "🏗️ 图层层级",
+        "Gandi/LayerManager@div2": "🔄 层级内顺序",
+        "Gandi/LayerManager@div3": "✨ 特效分层（配合雷神扩展）",
+        "Gandi/LayerManager@docsURI": "https://learn.ccw.site/article/ed40913d-60d6-4f81-95dc-bf05237aafeb",
+        "Gandi/LayerManager@setLayerManager": "[OP]图层管理器",
+        "Gandi/LayerManager@setSortOrder": "设置图层排序方式为[ORDER]",
+        "Gandi/LayerManager@checkLayerLevelsButton": "在开发者控制台打印图层层级结构",
+        "Gandi/LayerManager@infoHeader": "图层层级结构",
+        "Gandi/LayerManager@defineLayerLevels": "定义图层层级(由高到低)[LEVELS]，并以[LEVEL]作为默认层级",
+        "Gandi/LayerManager@defaultLevels": "UI层,默认层,背景层",
+        "Gandi/LayerManager@defaultLevel": "默认层",
+        "Gandi/LayerManager@levelUndefined": "还未定义层级，请先定义",
+        "Gandi/LayerManager@moveSpritesInFolderToLevel": "将文件夹[FOLDER]中的[TYPE]移到层级[LEVEL]",
+        "Gandi/LayerManager@onlySprites": "仅角色",
+        "Gandi/LayerManager@spritesAndClones": "角色和克隆体",
+        "Gandi/LayerManager@setTargetLayerLevel": "将[TARGET]移到层级[LEVEL]",
+        "Gandi/LayerManager@setTargetLayerPriority": "将[TARGET]的层级内排序值设为[PRIORITY]",
+        "Gandi/LayerManager@setCanvasLayerLevel": "🖼️设置[TARGET]的高级画笔的层级[LEVEL]",
+        "Gandi/LayerManager@setCanvasLayerPriority": "🖼️将[TARGET]的高级画笔的图层排序值设为[PRIORITY]",
+        "Gandi/LayerManager@getTargetLayerInfo": "[TARGET]的[INFO]",
+        "Gandi/LayerManager@my": "我",
+        "Gandi/LayerManager@myself": "我",
+        "Gandi/LayerManager@setMinUnshadedIndex": "使层级[LEVEL]中排序值[INDEX]及上方图层不受雷神特效影响",
+        "Gandi/LayerManager@setMinUnshadedTarget": "使角色[TARGET]及其上方图层不受雷神特效影响",
+        "Gandi/LayerManager@setMinUnshadedLevel": "使层级[LEVEL]及其上方图层不受雷神特效影响",
+        "Gandi/LayerManager@allSprites": "全体角色",
+        "Gandi/LayerManager@inFrontOf": "前面",
+        "Gandi/LayerManager@behind": "后面",
+        "Gandi/LayerManager@ascending": "升序",
+        "Gandi/LayerManager@descending": "降序",
+        "Gandi/LayerManager@activate": "启动",
+        "Gandi/LayerManager@deactivate": "关闭",
+        "Gandi/LayerManager@affected": "受",
+        "Gandi/LayerManager@unaffected": "不受",
+        "Gandi/LayerManager@level": "图层层级",
+        "Gandi/LayerManager@myPriority": "层级内排序值",
+        "Gandi/LayerManager@myLayer": "图层序号"
```

Comment:
I'm pretty sure extension translations are automatically fetched from Transifex, so this should be removed.

Reply:
Okay I will restore them.